components/more-less/more-less.js

```diff
@@ -231,7 +240,8 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 
 	__expand() {
 		this.__transitionAdded = true;
-		this.__contentHeight = `${this.__content.scrollHeight}px`;
+		this.__contentHeight = 'unset';
+		this.__maxHeight = 'unset';
```

Comment:
I think this should be set to `this.height` since the component starts out in "shrunken" form.